src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,153 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import './FindMovie.scss';
-
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie, movies }) => {
+  const [title, setTitle] = useState('');
+  const [isPreviewShown, switchPreviewDisplay] = useState(false);
+  const [newMovie, setNewMovie] = useState({});
+  const [error, setError] = useState('');
+  const [isAddButtonDisabled, switchAddButtonDisability] = useState(true);
+  const [isLoading, switchLoading] = useState(false);
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [isPreviewShown, setIsPreviewDisplay] = useState(false);
  const [newMovie, setNewMovie] = useState({});
  const [error, setError] = useState('');
  const [isAddButtonDisabled, setIsAddButtonDisability] = useState(true);
  const [isLoading, setIsLoading] = useState(false);
```
Обычно делают так, таким образом всегда понят что если это 'set' то это установка state